### PR TITLE
Added support for OpenVDB AX on Windows

### PR DESCRIPTION
**Pull request by Idclip**
Signed-off-by: Nick Avramoussis <4256455+Idclip@users.noreply.github.com>

### CODE REVIEW

**Review by danrbailey — APPROVED** [review]
Looks good. It would be great to find some time to convert the AX unit tests to GoogleTest.

**Comment by Idclip** [comment]
Thanks @danrbailey, agreed about CppUnit, though I realistically don't think I'll be able to prioritise that anytime soon :(

File: openvdb_ax/openvdb_ax/math/OpenSimplexNoise.h [hunk 1]
```diff
@@ -101,7 +101,7 @@ namespace OSN
  */
 
 // 3D Implementation of the OpenSimplexNoise generator.
-class OSNoise
+class OPENVDB_AX_API OSNoise
```
Comment (richhones) [inline openvdb_ax/openvdb_ax/math/OpenSimplexNoise.h:104]: was gonna comment about whether this is needed, i.e. whether we should export these symbols but i guess for backwards compatibility we should regardless of whether we want other people to use these noise functions outside of our needs in AX

Comment (Idclip) [reply openvdb_ax/openvdb_ax/math/OpenSimplexNoise.h:104]: Anything that's tested in the unit tests needs to be exported. I suspect we'll end up deprecating these classes anyway

**Pull request merged**